### PR TITLE
Add start & shutdown Hana APIs to swagger spec.

### PR DESCRIPTION
**Pull request by vipulha**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Review by lagalbra — APPROVED** [review]
You need to check the box in the PR description to "wait for ARM feedback" since you added new APIs in this PR.

**Review by PakDLiu — APPROVED** [review]

**Comment by amarzavery** [comment]
ARM team needs to sign-off since new APIs are added. From my perspective, the PR looks good.

**Review by amarzavery — APPROVED** [review]

File: specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2017-11-03-preview/hanaonazure.json
Comment (ravbhatnagar) [inline specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2017-11-03-preview/hanaonazure.json]: please model "default" response for all your APIs. Sample here - https://github.com/Azure/azure-rest-api-specs/blob/master/specification/managedservices/resource-manager/Microsoft.ManagedServices/stable/2019-06-01/managedservices.json#L62

**Review by ravbhatnagar — COMMENTED** [review]
@vipulha just one minor comment. please address.

**Comment by vipulha** [comment]
> @vipulha just one minor comment. please address.
Added default response.

**Comment by ravbhatnagar** [comment]
Signing off from ARM side.

**Pull request merged**
